client/scss/components/_switch.scss

```diff
@@ -0,0 +1,83 @@
+$switch-width: 40px;
+$switch-height: 20px;
+$switch-border: 2px;
+$switch-outline: 3px;
+
+$switch-border-radius: ($switch-height + $switch-border * 2) / 2;
+$switch-outline-radius: $switch-border-radius + $switch-outline;
+
+// All the greys in Wagtail are really dark or really bright
+$switch-color-middle-grey: #777777;
```

Comment:
This feels like it could be useful more generally - should we add it to global grey colour variables instead?

Reply:
Would be nice to, not sure what the most consistent way to do this would be though, the current greys are numbered 1 - 5 so I guess we would either need to shift some of them or come up with a new way to label them